src/ol_concourse/pipelines/libraries/unified_ecommerce_api_clients.py

```diff
@@ -65,7 +65,7 @@ def _read_script(script_name: str) -> str:
 unified_ecommerce_repository = git_repo(
     name=Identifier("unified-ecommerce"),
     uri=unified_ecommerce_repository_uri,
-    branch="release",
+    branch="main",
```

Comment:
Question about this - changing this will mean that the API clients rebuild whenever we push to `main`? I'm not sure we want to do that since `main` won't necessarily be production code.

Reply:
This is mostly because the `release` branch wasn't there when I started looking at this early last week. Seems like it should be changed back now that it is available.